pkgs/development/tools/knightos/kimg/default.nix

```diff
@@ -11,7 +11,8 @@ stdenv.mkDerivation rec {
     sha256 = "040782k3rh2a5mhbfgr9gnbfis0wgxvi27vhfn7l35vrr12sw1l3";
   };
 
-  nativeBuildInputs = [ cmake asciidoc ];
+  strictDeps = true;
+  nativeBuildInputs = [ cmake libxslt.bin asciidoc ];
```

Comment:
Formatting nitpick: it would be nice to have a blank line between `strictDeps` and `nativeBuildInputs`. Also, it's nicer to sort the dependencies alphabetically in `.*[bB]uildInputs`.

Reply:
Done.